src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
That's not how it works though? Did you see the reasoning for having a unique event per "loading" widget? Each symbol creates its own event, so you can have `const foo = Symbol()` & `const bar = Symbol()` and use both as registry labels for a widget that is not "loaded" and all registry handlers that had called `.get(foo)` or `.get(bar)` will be told when that widget is loaded as the registry emits `.emit({ type: foo })` or `.emit({ type: bar });`?